src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpStackOverFlowTests.cs

```diff
@@ -55,10 +48,14 @@ public async Task TestSyntaxIndex()
             // Call FAR to create syntax index. The goal is to verify we don't hit StackOverFlow during the creation.
             await TestServices.Input.SendAsync((VirtualKeyCode.F12, VirtualKeyCode.SHIFT), HangMitigatingCancellationToken);
             var contents = await TestServices.FindReferencesWindow.GetContentsAsync(HangMitigatingCancellationToken);
-            Assert.Single(contents);
-            var content = contents.Single();
-            content.TryGetValue(StandardTableKeyNames.Text, out string code);
-            Assert.Equal("int Tree82(int i)", code);
+            Assert.Equal(18, contents.Length);
```

Comment:
This is the real reference count in the sample file. This will bring it to the real expected behavior. (Even though I still can't get it passed in CI)
The previous Assert.Single() is because when our FAR engine fetch result one by one, so when I wrote the test, it get one result, and then crash.

No real impact since this test is still skipped.